index.js

```diff
@@ -168,7 +170,8 @@ module.exports = {
             `apostrophe-i18n:${argv.locale}`,
             async () => fs.writeJson(file, translations, { spaces: 2 })
           );
-          console.timeEnd(`${argv.locale} done in`);
+
+          argv.verbose && console.timeEnd(`${argv.locale} done in`);
```

Comment:
Please put this in an if statement for consistency with our coding style.

Reply:
Done